doc/releases/changelog-0.41.0.md

```diff
@@ -420,29 +420,40 @@ With `qml.decompositions.enable_graph()`, the following new features are availab
 
 <h4>QROM State Preparation 📖</h4>
 
-* Added template `qml.QROMStatePreparation` that prepares arbitrary states using `qml.QROM`.
+Quantum Read-Only Memory (QROM) isn't just useful for loading bitstrings.
+It can be leveraged to prepare entire quantum states!
```

Comment:
This is probably not dry enough for the changelog 😆. Could put this in the blog, though! 👀